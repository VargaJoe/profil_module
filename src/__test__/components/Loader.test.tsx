import * as React 						from 'react';
import Loader 							from '../../components/Loader';
import { 
	configure, 
    shallow,
    mount } 							from 'enzyme';
import * as Adapter 					from 'enzyme-adapter-react-16';
import toJson 				      		from 'enzyme-to-json';

configure( {adapter: new Adapter()} );

describe('<Loader /> shallow rendering', () => {
    
    const loader = shallow(<Loader />); 
    
    // test Snapshot 
	it('Match to snapshot', () => {
		expect(toJson(loader)).toMatchSnapshot();
    });
    
});
 