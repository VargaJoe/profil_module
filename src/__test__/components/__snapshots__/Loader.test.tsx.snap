// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> shallow rendering Match to snapshot 1`] = `
<div
  className="loader"
>
  <div
    className="leftEye"
  />
  <div
    className="rightEye"
  />
  <div
    className="mouth"
  />
</div>
`;
